001_skim_lit_project_v1_0.ipynb

```diff
@@ -0,0 +1,195 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "GgCl01YSjggZ"
+      },
+      "source": [
+        ""
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "s7wYp3FPkI-Z"
+      },
+      "source": [
+        "1. **Get Data**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "F5z1nIfMkLsM",
+        "outputId": "ff7c475e-7f58-4dd0-d0a4-43787192c0d6"
+      },
+      "source": [
+        "!git clone https://github.com/Franck-Dernoncourt/pubmed-rct\n",
+        "!ls pubmed-rct"
+      ],
+      "execution_count": 1,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Cloning into 'pubmed-rct'...\n",
+            "remote: Enumerating objects: 33, done.\u001b[K\n",
+            "remote: Counting objects: 100% (3/3), done.\u001b[K\n",
+            "remote: Compressing objects: 100% (3/3), done.\u001b[K\n",
+            "remote: Total 33 (delta 0), reused 0 (delta 0), pack-reused 30\u001b[K\n",
+            "Unpacking objects: 100% (33/33), done.\n",
+            "PubMed_200k_RCT\n",
+            "PubMed_200k_RCT_numbers_replaced_with_at_sign\n",
+            "PubMed_20k_RCT\n",
+            "PubMed_20k_RCT_numbers_replaced_with_at_sign\n",
+            "README.md\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "X2t6-aZAkP3w",
+        "outputId": "bf97fb99-dfc2-465a-a686-a6d1839a5008"
+      },
+      "source": [
+        "!ls pubmed-rct/PubMed_20k_RCT_numbers_replaced_with_at_sign/"
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "dev.txt  test.txt  train.txt\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "A8ghc1xEl0vR",
+        "outputId": "fb5074e5-2888-448a-df36-8cd7ac8a8d3e"
+      },
+      "source": [
+        "!ls pubmed-rct/PubMed_20k_RCT_numbers_replaced_with_at_sign/"
+      ],
+      "execution_count": 3,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "dev.txt  test.txt  train.txt\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "bYsmxzL6nqhY",
+        "outputId": "90642599-542f-4244-bfd2-432088fb83c6"
+      },
+      "source": [
+        "data_dir = \"/content/pubmed-rct/PubMed_20k_RCT_numbers_replaced_with_at_sign/\"\n",
```

Comment:
Can you do something so that you don't need to download training data again and again.